XCUITests/IntegrationTests.swift

```diff
@@ -44,4 +44,31 @@ class IntegrationTests: BaseTestCase {
         navigator.goto(SettingsScreen)
         waitforExistence(app.tables.staticTexts["Sync Now"], timeout: 10)
     }
+
+    func testFxASyncTabs () {
+        navigator.openURL("example.com")
+        waitUntilPageLoad()
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"], timeout: 10)
+        userState.fxaUsername = ProcessInfo.processInfo.environment["FXA_EMAIL"]!
+        userState.fxaPassword = ProcessInfo.processInfo.environment["FXA_PASSWORD"]!
+        navigator.performAction(Action.FxATypeEmail)
+        navigator.performAction(Action.FxATypePassword)
+        navigator.performAction(Action.FxATapOnSignInButton)
+        allowNotifications()
+
+        // Wait for initial sync to complete
+        navigator.nowAt(BrowserTab)
+        // This is only to check that the device's name changed
+        navigator.goto(SettingsScreen)
+        app.tables.cells.element(boundBy: 0).tap()
+        waitforExistence(app.cells["DeviceNameSetting"].textFields["DeviceNameSettingTextField"])
+        XCTAssertEqual(app.cells["DeviceNameSetting"].textFields["DeviceNameSettingTextField"].value! as! String, "Fennec on iOS")
+
+        // Sync again just to make sure to sync after new name is shown
```

Comment:
Curious, is this affecting the test that comes after this? We don't do anything after below sync operation, what's the purpose of this sync again?

Reply:
So we had an issue before that could not see the change on desktop and we were asked if we were sure that there was a sync after the name was changed so that it was reflected on desktop, that is why I added this. But to be honest, not sure if changing the name using LaunchArg makes this unnecessary since the name is the same from the beginning and once there is a sync before, should be enough...